### PR TITLE
Fix Bug 1146683 - The php_url helper function is broken

### PR DESCRIPTION
**Pull request by ishanroy**

### CODE REVIEW

File: bedrock/mozorg/helpers/misc.py [hunk 1]
```diff
@@ -35,17 +35,6 @@ def convert_to_high_res(url):
...
-@jinja2.contextfunction
-def php_url(ctx, url):
-    """Process a URL on the PHP site and prefix the locale to it."""
-    locale = getattr(ctx['request'], 'locale', None)
-
-    # Do this only if we have a locale and the URL is absolute
-    if locale and url[0] == '/':
-        return path.join('/', locale, url.lstrip('/'))
-    return url
-
```
Comment (jpetto) [inline bedrock/mozorg/helpers/misc.py]: There are 1 too many blank lines between functions here, which is why the tests aren't passing. There should be two blank lines in-between each function.

**Comment by jpetto** [comment]
Looks like there's one more use of `php_url` in `bedrock/foundation/templates/foundation/trademarks/policy.html` on [line 82](https://github.com/mozilla/bedrock/blob/master/bedrock/foundation/templates/foundation/trademarks/policy.html#L82) that needs to be updated.

**Comment by jpetto** [comment]
Thanks for taking this on @ishanroy!

In addition to the two tasks requested above, we need to complete one more thing to finalize this pull request. To keep the git history clean, [squash your commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) and update your commit message to reference the bug number and description. For this PR, I would recommend a commit message like:

`[fix bug 1146683] Remove php_url helper function.`

If you need any assistance squashing your commits, just comment here and someone will help you along.

One final note - it's generally best to create a separate branch when working on a bug. For example, if you look at PR #2986, you can see the developer created a branch named after the bug (`bug-1162048-use-mozid-modal`). You don't have to create a new branch for this PR, but keep this approach in mind for future pull requests. :smile:

**Comment by ishanroy** [comment]
Thanks for your help @jpetto. Will keep in mind to work on branches in future.

**Comment by jpetto** [comment]
This change is looking great @ishanroy. Thanks!

r+ :cake:

**Comment by ishanroy** [comment]
Thanks @jpetto. Was my first ever bug. Happy to help!

**Pull request merged**